src/Mutator/Boolean/TrueValue.php

```diff
@@ -92,8 +92,8 @@ public function canMutate(Node $node): bool
             return false;
         }
 
-        $parentNode = $node->getAttribute(ParentConnectorVisitor::PARENT_KEY);
-        $grandParentNode = $parentNode !== null ? $parentNode->getAttribute(ParentConnectorVisitor::PARENT_KEY) : null;
+        $parentNode = ParentConnector::findParent($node);
+        $grandParentNode = $parentNode !== null ? ParentConnector::findParent($parentNode) : null;
```

Comment:
`if ($parentNode === null) return null; ...` ?

Reply:
it was a ternary before, I don't really fancy replacing it; If we want to get rid of ternary operators it would be better to replace them everywhere IMO

Reply:
Fair enough.